Sources/Models/Views/ScrollViews/CollectionView.swift

```diff
@@ -302,6 +303,19 @@ public struct CollectionViewCell: IBDecodable, ViewProtocol, IBIdentifiable, IBR
     }
 }
 
+class ToggleKey {
```

Comment:
Umm, `ToggleKey` seems very dependent on order of how container decode the view.
And I think there will be more version dependent decode operation in the future, so I want to avoid this kind of complex idiom.
Could you consider having IB version in indexer and switch the key by version?

Reply:
Yes, like we discuss in the issue. 
I could not do it immediately, that's why I make this dirty fix 🥴

Reply:
I'm thinking use of IB version is not good idea because existing xml key will not automatically changed by Xcode 11.

- Create collection view cell at XIB file in Xcode 10
- Open it in Xcode 11
- Add one more collection view cell, result is `view` and `collectionViewCellContentView` keys mixed xml like this.

```xml
<collectionViewCell opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" id="uzK-LR-Mxj">
    <rect key="frame" x="0.0" y="0.0" width="50" height="50"/>
    <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
    <view key="contentView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" insetsLayoutMarginsFromSafeArea="NO">
        <rect key="frame" x="0.0" y="0.0" width="50" height="50"/>
        <autoresizingMask key="autoresizingMask"/>
    </view>
    <viewLayoutGuide key="safeArea" id="7dg-fX-HkR"/>
    <point key="canvasLocation" x="-2" y="58"/>
</collectionViewCell>
<collectionViewCell opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" id="LEX-Ny-2b1">
    <rect key="frame" x="0.0" y="0.0" width="50" height="50"/>
    <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
    <collectionViewCellContentView key="contentView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" insetsLayoutMarginsFromSafeArea="NO" id="BjQ-7p-u3U">
        <rect key="frame" x="0.0" y="0.0" width="50" height="50"/>
        <autoresizingMask key="autoresizingMask"/>
    </collectionViewCellContentView>
    <point key="canvasLocation" x="32" y="234"/>
</collectionViewCell>
```

Reply:
ok so testing the two key is the only solution
but maybe we could find a more elegant way